src/main/kotlin/com/sksamuel/avro4k/schema/ClassSchemaFor.kt

```diff
@@ -50,18 +56,29 @@ class ClassSchemaFor(
    }
 
    private fun dataClassSchema(): Schema {
+      val fullName = "${naming.namespace()}.${naming.name()}"
```

Comment:
How about adding a property qualifiedName to RecordNaming?

Reply:
ok